fhir-path/src/main/java/org/linuxforhealth/fhir/path/FHIRPathQuantityNode.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright IBM Corp. 2019, 2021
+ * (C) Copyright IBM Corp. 2019, 2021, 2022
```

Comment:
we use just 2 dates.
1. the initial date
2. the last modified date
```suggestion
 * (C) Copyright IBM Corp. 2019, 2022
```

Reply:
This has been fixed.